src/azure-cli/azure/cli/command_modules/sql/commands.py

```diff
@@ -260,9 +260,9 @@ def load_command_table(self, _):
                             database_blob_auditing_policies_operations,
                             client_factory=get_sql_database_blob_auditing_policies_operations) as g:
 
-        g.show_command('show', 'get')
-        g.generic_update_command('update',
-                                 custom_func_name='db_audit_policy_update')
+        g.custom_show_command('show', 'db_audit_policy_show')
+        g.custom_command('update', 'db_audit_policy_update')
```

Comment:
May I know the reason changing generic_update_command to custom_command? With this change, you will lose some parameters like --set, --add, --remove.